drivers/SmartThings/zigbee-contact/profiles/contact-battery-temperature-tamper.yml

```diff
@@ -0,0 +1,30 @@
+name: contact-battery-temperature-tamper
+components:
+- id: main
+  capabilities:
+  - id: contactSensor
+    version: 1
+  - id: battery
+    version: 1
+  - id: temperatureMeasurement
+    version: 1
+  - id: tamperAlert
+    version: 1
+  - id: firmwareUpdate
+    version: 1
+  - id: refresh
+    version: 1
+  categories:
+  - name: ContactSensor
+preferences:
+  - preferenceId: tempOffset
+    explicit: true
+  - title: "Temperature Sensitivity (°)"
```

Comment:
if this is Celsius, could you indicate that? Some users may have their systems set to Fahrenheit